src/components/TabMenu/TabMenu.tsx

```diff
@@ -43,9 +44,17 @@ export const EasyTabMenu: React.FC<EasyTabMenuProps> = ({
   size = 'md',
   ...props
 }) => {
+  const safeTabs: EasyTabType[] = [];
+
+  tabs.forEach((tab) => {
+    if (tab) {
+      safeTabs.push(tab);
+    }
+  });
```

Comment:
might be simpler logic to filter by truthyness
```
tabs.filter(tab => !!tab)
.map..
```

Reply:
I tried, but TS didn't like turning a list of `EasyTabType | undefined` tab into a list of `EasyTabType`, even with the filtering.

Maybe I was missing something obvious?

![image](https://user-images.githubusercontent.com/304177/92166948-26b41180-edff-11ea-9c10-9b1b1df008f4.png)

Reply:
BTW, I tried doing it inline with the `.map` first, then broke it out into its own const to verify what was going on.

Reply:
Ah, I'm not the first! Found a maybe better way.
https://stackoverflow.com/a/46700791

Reply:
oh man, sorry for the TS rabbit hole